src/app/home/home.component.scss

```
@@ -1,107 +1,101 @@
+@import "../_variables.scss";
 
-.greetings {
-  margin-top: 30px;
-  margin-bottom: 30px;
-}
-
-.greetings-hero-img {
-  position: center;
-  height: 80%;
-  width: 80%;
-  border-radius: 50%;
-}
-
-.mobile-greetings-hero-img {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  position: center;
-  height: 40%;
-  width: 40%;
-  border-radius: 50%;
-}
-
-.nomargin {
-  margin: 0;
-}
-
-.grid-container {
-  margin-top: 20px;
-  margin-bottom: 20px;
+.background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: $black;
+  z-index: -999; // hacky way to override app background color for now
 }
 
-.dashboard-card {
-  position: absolute;
-  top: 15px;
-  bottom: 15px;
+.landing-page {
+  background: $black;
+  font-family: Roboto, sans-serif;
+  text-align: center;
+  color: white;
 }
 
-.dashboard-card mat-card {
-  height: 95%;
+// mobile default
+.ddb-logo {
+  margin: 37px 0 6px;
+  width: 90px;
 }
 
-.card-center {
-  left: 5px;
-  right: 5px;
+.avrae-logo {
+  margin: 6px 0 21px;
+  width: 177px;
 }
 
-.card-left {
-  left: 0;
-  right: 10px;
+.about-avrae {
+  margin: 0 29px 39px;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 140%;
```

Using any unit for line-height can cause strange results depending on the font size of parents or the element. I thought that percents would be fine, since they are basically just a ratio multiplied by 100, but the MDN seems to think they can cause issues as well. I always use a unitless scalar value when setting line-height to avoid the edge cases that can sometimes crop up.

```suggestion
  line-height: 1.4;
```

From the MDN:
> \<number\> (unitless)
> The used value is this unitless <number> multiplied by the element's own font size. The computed value is the same as the specified <number>. In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance.

> \<percentage\>
> Relative to the font size of the element itself. The computed value is this <percentage> multiplied by the element's computed font size. Percentage values may produce unexpected results (see the second example below).

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height